### ISSUE TITLE
`Select` component accessibility - Elements must meet minimum colour contrast ratio thresholds

### ISSUE THREAD

**Issue by thedannygoodall** [Sage/carbon#6722]
### Description

There is an issue regarding colour contrast with the `Select` component when it is disabled. 
> Element has insufficient color contrast of 2.08 (foreground color: #a9acac, background color: #f2f5f6, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

### Reproduction

https://carbon.sage.com/?path=/story/select--disabled

### Steps to reproduce

1. Implement a `Select` component
2. Set the `disabled` property to `true`

### Carbon version

134.0.0

### Relevant browsers

Chrome

### Relevant OSs

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nineteen88** [comment]
@ljemmo @harpalsingh  My understanding of this issue is that we don't actively encourage using Disabled form components and they are an edge case that doesn't support colour contrast ratios. From the DS perspective should a pattern be prescribed to avoid this (like conditionally render or use readOnly instead) or do we just accept the colour contrast issues?

**Comment by ljemmo** [comment]
@nineteen88 On our end, we'll be putting messaging on the DS pages to discourage use of disabled states. However, we recognise there is the odd edge case for their use so there is still a need to support them. AFAIK we just accept the contrast issues as a WCAG exception for now. I will share this thread with Andy (accessibility speciailist) to see if thats his steer.

**Comment by v99rly** [comment]
@nineteen88 @ljemmo @harpalsingh Hey Guys, We should actively avoid using disabled states unless absolutely necessary. We don't provide guidance around this to enforce the "avoid using disabled states" narrative. As @ljemmo mentioned we will update the guidance to show this, if there is a valid use case for a disabled state and all other avenues have been explored then we will have to to accept the failure on contrast ratios for now.

**Comment by edleeks87** [comment]
Thanks for all the input on this, based on the above comments I'm going to close the issue
